WordPress/Classes/Extensions/NSString+Helpers.swift

```diff
@@ -3,9 +3,12 @@ import Foundation
 
 extension NSString
 {
-    /**
-     *  @details Splits the lines contained in the current string, and returns its unique values in a NSSet instance.
-     */
+    /// Returns the string's hostname. If there's any
+    public func hostname() -> String {
+        return NSURLComponents(string: self as String)?.host ?? String()
+    }
```

Comment:
I marked the method as deprecated and only got those two warnings. Are you sure it's still being used somewhere else?